openreview/venue_request/process/matching_pre_process.py

```diff
@@ -1,4 +1,20 @@
 def process(client, note, invitation):
+    _, baseurl_v2 = openreview.tools.get_base_urls(client)
+    client_v2 = openreview.api.OpenReviewClient(baseurl=baseurl_v2, token=client.token)
+    forum_note = client.get_note(note.forum)
+    venue_id = forum_note.content.get('venue_id', '')
+    domain = client_v2.get_group(venue_id)
+    submission_venue_id = domain.content['submission_venue_id']['value']
+    senior_area_chairs_name = domain.content.get('senior_area_chairs_name', {}).get('value')
+    compute_affinity_scores = note.content.get('compute_affinity_scores') == 'Yes'
 
-    if 'Yes' in note.content['compute_affinity_scores'] and 'upload_affinity_scores' in note.content:
-        raise openreview.OpenReviewException('Either upload your own affinity scores or select affinity scores computed by OpenReview')
\ No newline at end of file
+    if compute_affinity_scores and 'upload_affinity_scores' in note.content:
```

Comment:
I guess you are missing `not compute_affinity_scores` is it is reading the yes value.